debian/changelog

```diff
@@ -1,8 +1,14 @@
-pop-gtk-theme (5.3.1) UNRELEASED; urgency=medium
+pop-gtk-theme (5.3.2) groovy; urgency=medium
+
+  * Make calendar labels more legible, especially in the dark mode
+
+ -- Ian Santopietro <ian@system76.com>  Wed, 18 Nov 2020 14:36:36 -0700
+
+pop-gtk-theme (5.3.1) groovy; urgency=medium
 
   * Add styling for Nautilus Path Bars
 
- -- Ian Santopietro <ian@system76.com>  Thu, 12 Nov 2020 17:14:27 -0700
+ -- Ian Santopietro <ian@system76.com>  Wed, 18 Nov 2020 14:35:52 -0700
```

Comment:
This line probably shouldn't change.

Reply:
Yep, sometimes dch is a jerk. Fixed!

Reply:
Indeed. Like `debhelper` it seems to sometimes be a little too helpful...